server/parsec/logging.py

```diff
@@ -181,32 +171,52 @@ def _structlog_to_sentry_processor(
         if exc_info:
             sentry_event, _ = event_from_exception(
                 exc_info=exc_info,
-                client_options=sentry_client.options,
+                client_options=sentry_sdk.get_client().options,
```

Comment:
Are you sure it's ok to call `get_client()` for each log? just checking :)

Reply:
Not sure what you mean... are you suggesting that it might expensive to call it?

The client is retrieved from the current scope, and I do not see anything in their code that can be expensive. https://github.com/getsentry/sentry-python/blob/e6ca5a28dd139097ad7c8cb468e0b9232185b728/sentry_sdk/scope.py#L372

Also, keep in mind that this `get_client()` call is only executed when an Exception occurs AND the message contains the `exc_info`. That shouldn't happen too often, right?